src/components/notification/__tests__/NotificationBadge.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { mount } from 'enzyme';
+import NotificationBadge from '../NotificationBadge';
+import { Notification } from '../notificationShape';
+
+const studentNotifications: Notification[] = [
+  {
+    id: 1,
+    type: 'new',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assessment_title: 'The Secret to Streams'
+  },
+  {
+    id: 2,
+    type: 'graded',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assessment_title: 'The Secret to Streams'
+  },
+  {
+    id: 3,
+    type: 'autograded',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assessment_title: 'The Secret to Streams'
+  },
+  {
+    id: 4,
+    type: 'unsubmitted',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assessment_title: 'The Secret to Streams'
+  }
+];
+
+const avengerNotifications: Notification[] = [
+  {
+    id: 1,
+    type: 'submitted',
+    assessment_id: 1,
+    assessment_type: 'Mission',
+    assessment_title: 'The Secret to Streams',
+    submission_id: 1
+  }
+];
+
+describe('Badge', () => {
+  test('renders properly for students', () => {
+    const tree = mount(<NotificationBadge number={studentNotifications.length} />);
+
+    expect(tree.debug()).toMatchSnapshot();
+  });
+
+  test('renders properly for avengers', () => {
```

Comment:
Why did you split by avengers and student? The component has no logic to differentiate between the roles?

I feel these tests are redundant (Blueprint have already probably tested their `Tag` component) There is no need for us to retest it here.